package.json

```diff
@@ -98,13 +98,14 @@
   "dependencies": {
     "@across-protocol/across-token": "^1.0.0",
     "@across-protocol/constants-v2": "^1.0.9",
-    "@across-protocol/contracts-v2": "^2.4.7",
+    "@across-protocol/contracts-v2": "^2.4.8",
```

Comment:
For reference, the delta from v2.4.7 to v2.4.8 is this commit: https://github.com/across-protocol/contracts-v2/commit/65a007c5e6dbb51b1a54b770f246d2002dafb1b0